connector_alndata/models/res_partner_industry.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2019 Open Source Integrators
+# Copyright (C) 2019 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResPartnerIndustry(models.Model):
+    _inherit = "res.partner.industry"
+
+    ref = fields.Char('Reference')
```

Comment:
@nikitavaghela We are missing the unique constraint.

Reply:
done